src/constants.py

```diff
@@ -1,4 +1,53 @@
+ADD_LABELS = "add_labels"
+REMOVE_LABELS = "remove_labels"
+
+
+def title_labels(labels):
+    return [label.title() for label in labels]
+
+
+def title_user_input_map_labels(user_input_map):
+    all_add_labels = []
+    for key in user_input_map:
+        user_input_map[key][ADD_LABELS] = title_labels(user_input_map[key][ADD_LABELS])
+        all_add_labels.extend(user_input_map[key][ADD_LABELS])
+
+    return set(all_add_labels)
+
+
+def create_remove_labels_in_user_input_map(user_input_map, all_add_labels):
+    for key in user_input_map:
+        user_input_map[key][REMOVE_LABELS] = list(
+            all_add_labels - set(user_input_map[key][ADD_LABELS])
+        )
+
+
 BLOCK_MERGE_VERIFY_CONTEXT = "Verified label"
+
 LABEL_VERIFIED = "Verified"
 LABEL_APPROVE = "Approve"
 NEEDS_MAINTAINERS_APPROVE = "Needs Maintainers Approve"
+
+SEMVER = "semver"
+
+LABEL_KEY_MAJOR = "major"
+LABEL_KEY_MINOR = "minor"
+LABEL_KEY_PATCH = "patch"
+
+BLOCK_MERGE_SEMVER_CONTEXT = "Semver label"
+SEMVER_USER_INPUT_LABELS_MAP = {
```

Comment:
This will do the same with less code and in my opinion it's more readable
```python
SEMVER_USER_INPUT_LABELS_MAP = {
    LABEL_KEY_MAJOR: f"{LABEL_KEY_MAJOR.title()}/Breaking-Change",
    LABEL_KEY_MINOR: f"{LABEL_KEY_MINOR.title()}/Non-Breaking-Feature",
    LABEL_KEY_PATCH: f"{LABEL_KEY_PATCH.title()}/Non-Breaking-Fix",
}
SEMVER_LABELS_LIST = list(SEMVER_USER_INPUT_LABELS_MAP.values())

if __name__ == "__main__":
    user_label = "major"
    label_to_add = SEMVER_USER_INPUT_LABELS_MAP["major"]
    labels_to_remove = set(SEMVER_LABELS_LIST) - {SEMVER_USER_INPUT_LABELS_MAP["major"]}
```

output:
```
ipdb> label_to_add
'Major/Breaking-Change'
ipdb> labels_to_remove
{'Minor/Non-Breaking-Feature', 'Patch/Non-Breaking-Fix'}
```